install.sh

```diff
@@ -226,6 +243,7 @@ nvm_do_install() {
     fi
   fi
 
+  nvm_bash_completion
```

Comment:
Currently it's not - the install script tries to guess which one is appropriate, and picks only one, via the `nvm_detect_profile` method - perhaps you could use that method in `nvm_bash_completion`, and use `case` to determine if it's one of the three that would be supported?